lib/ansible/modules/rpm_key.py

```diff
@@ -164,30 +167,42 @@ def fetch_key(self, url):
         tmpfile.close()
         return tmpname
 
+    def normalize_keyids(self, keyids):
+        normalized_keyids = []
+        for keyid in keyids:
+            normalized_keyids.append(self.normalize_keyid(keyid))
+
+        return normalized_keyids
+
     def normalize_keyid(self, keyid):
         """Ensure a keyid doesn't have a leading 0x, has leading or trailing whitespace, and make sure is uppercase"""
         ret = keyid.strip().upper()
-        if ret.startswith('0x'):
+        if ret.startswith("0x"):
```

Comment:
*Pro tip:* mixing formatting changes with functional ones is discouraged because it makes it harder for the reviewers to identify which places in the patch are useful payload. These should always be separated.